README.md

```diff
@@ -565,14 +609,9 @@ Anything that enables or disables plugins on runtime is extremely dangerous. Loa
 
 ## mspt
 Paper offers a `/mspt` command that will tell you how much time the server took to calculate recent ticks. If the first and second value you see are lower than 50, then congratulations! Your server is not lagging! If the third value is over 50 then it means there was at least 1 tick that took longer. That's completely normal and happens from time to time, so don't panic.
-
-## timings
```

Comment:
Timings is still relevant on paper. Change the section to mention that spark should be used, but leave the mention of it